pkg/stub/jenkins.go

```diff
@@ -15,6 +15,7 @@ func tagInPayload(tag, env string, stream *imagev1.ImageStream) *imagev1.ImageSt
 	}
 	for _, tagSpec := range stream.Spec.Tags {
 		if tagSpec.Name == tag {
+			logrus.Printf("updating image ref for tag %s in stream %s with image %s", tag, stream.Name, imageRef)
```

Comment:
should be debug level?

Reply:
actually in the end I consciously chose non debug level ... I'd liked having this information in there

these messages are infrequent compared to the other messaging

still want me to make it debug?

Reply:
falls under my statement from a few weeks ago that at some point we need to audit our default log messages, but for now it's ok.